EIPS/eip-7707.md

```diff
@@ -0,0 +1,147 @@
+---
+eip: 7707
+title: Incentivize Access List Provisioning
+description: This EIP proposes updating gas cost parameters for access lists to incentivise their use and improve transaction execution efficiency.
+author: Ben Adams (@benaadams), Oleg Iakushkin (@OlegJakushkin)
+discussions-to: https://ethereum-magicians.org/t/eip-7707-align-incentives-for-access-list-provisioning/20025
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-05-12
+requires: 2930
+---
+
+## Abstract
+
+To facilitate future developments of parallel data load, we need to
+encourage as many transactions as possible that contain as complete and
+valid access lists. Current access list pricing does not sufficiently incentivize their inclusion, and this EIP aims to address this gap.
```

Comment:
Abstract should contain a short human-readable description of the solution, not of the problem.  Motivation is where you describe the problem.

Reply:
Example for this EIP: "Reduces gas cost for storage reads from `a` and `b` to `c` and `d` when an access list is provided."

Reply:
Updated, please take a look.